src/stirling/source_connectors/stirling_error/stirling_error_bpf_test.cc

```diff
@@ -479,6 +471,11 @@ std::filesystem::path BazelJavaTestAppPath(const std::string_view app_name) {
 }  // namespace
 
 TEST_F(StirlingErrorTest, PerfProfilerNoPreserveFramePointer) {
+  if (std::getenv("TESTING_UNDER_QEMU") != nullptr) {
+    // This test crashes under qemu for unknown reasons, with a dcheck failure of:
+    // error_code == kPerfProfilerStatusOk (1 vs 0)
+    GTEST_SKIP() << "Skipping this test under qemu";
```

Comment:
This likely means that the records aren't being processed quickly enough. We can take a deeper look into why that might be.

Reply:
Leave a TODO here for someone to investigate.